### PR TITLE
create Pipfile and some directories.

### PR DESCRIPTION
**Pull request by noobOriented**
創建環境

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[[source]]
+verify_ssl = true
+name = "pypi"
+url = "https://pypi.org/simple"
+
+[packages]
+pytest = "*"
```
Comment (stegben) [inline Pipfile]: should be in dev-packages

```
pipenv install pytest --dev
```

**Review by stegben — CHANGES_REQUESTED** [review]
其他 LGTM

File: Pipfile (hunk 1 above)
Comment (noobOriented) [reply Pipfile]: 681387ef74de1f8d37f40acc18e7da8438ac8e8c done

**Review by stegben — APPROVED** [review]
LGTM

**Pull request merged**